domains/ckateptb.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "CKATEPTb",
+        "email": "taksis@ex.ua"
+      },
+
+      "record": {
+        "A": ["65.108.238.23"]
```

Comment:
404?

Reply:
> 404?

I want to host a maven repo and a business card on this domain in the future.

Reply:
@CKATEPTb We cannot give you the domain until there is content.

Reply:
@WilliamDavidHarrison fixed

Reply:
@CKATEPTb What port?

Reply:
> @CKATEPTb What port?

8080